src/hooks/errors.js

```diff
@@ -0,0 +1,20 @@
+import { SINGLE_ERROR_CLEAR } from 'actions/actionTypes';
+import { useDispatch, useSelector } from 'react-redux';
+import { typeWithoutSuffix } from 'selectors/selectorUtils';
+
+export const useError = (errorType) => {
+  const dispatch = useDispatch();
+
+  return useSelector((state) => {
+    const errorMessage = state.errors[typeWithoutSuffix(errorType)] ?? null;
```

Comment:
We can reuse our existing selector here, to not duplicate logic. It would look something like:
```javascript
const errorMessage = useSelector((state) => selectErrorByAction(state, errorType));
```

Reply:
ok! done!